setup.py

```diff
@@ -18,7 +18,7 @@
     long_description = open('README.md').read(),
     zip_safe         = False,
     include_package_data = True,
-    install_requires = ['robotframework'],
+    install_requires = ['robotframework>=3.2'],
```

Comment:
I would suggest putting at least version 3.2.1 because 3.2.0 should be avoided according to the official RF release notes.
```suggestion
    install_requires = ['robotframework>=3.2.1'],
```